src/Symfony/Component/Messenger/Event/HandlerStartingEvent.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Messenger\Event;
+
+/**
+ * Event dispatched before a handler is called.
+ */
+final class HandlerStartingEvent extends AbstractHandlerEvent
```

Comment:
The name of the class does not make it clear, that it is dispatched before.

Maybe BeforeHandlerStartEvent?

Reply:
I actually thought about this, but wanted to keep things organized by prefix, like `Worker*` events. I don't mind though, what about just `BeforeHandlerEvent`?

Reply:
> The name of the class does not make it clear, that it is dispatched before.

i dont see how it could mean something else